src/sentry/models/dashboard.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+from jsonfield import JSONField
+
+from sentry.constants import ObjectStatus
+from sentry.db.models import BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+
+
+class Dashboard(Model):
+    """
+    A dashboard.
+    """
+    __core__ = True
+
+    title = models.CharField(max_length=128)
+    owner = FlexibleForeignKey('sentry.User')
+    organization = FlexibleForeignKey('sentry.Organization')
+    data = JSONField(default={})
```

Comment:
Will we have more formal schemas for this data in the application code? Are there parts of that internals schema that could be promoted to the table schema?